karapace/protobuf/compare_result.py

```diff
@@ -0,0 +1,79 @@
+from enum import auto, Enum
+
+
+class Modification(Enum):
+    # TODO
+    PACKAGE_ALTER = auto()
+    SYNTAX_ALTER = auto()
+    MESSAGE_ADD = auto()
+    MESSAGE_DROP = auto()
+    MESSAGE_MOVE = auto()
+    ENUM_CONSTANT_ADD = auto()
+    ENUM_CONSTANT_ALTER = auto()
+    ENUM_CONSTANT_DROP = auto()
+    ENUM_ADD = auto()
+    ENUM_DROP = auto()
+    TYPE_ALTER = auto()
+    FIELD_ADD = auto()
+    FIELD_DROP = auto()
+    FIELD_MOVE = auto()
+    FIELD_LABEL_ALTER = auto()
+    FIELD_NAME_ALTER = auto()
+    FIELD_KIND_ALTER = auto()
+    FIELD_TYPE_ALTER = auto()
+    ONE_OF_ADD = auto()
+    ONE_OF_DROP = auto()
+    ONE_OF_MOVE = auto()
+    ONE_OF_FIELD_ADD = auto()
+    ONE_OF_FIELD_DROP = auto()
+    ONE_OF_FIELD_MOVE = auto()
+    FEW_FIELDS_CONVERTED_TO_ONE_OF = auto()
+
+    # protobuf compatibility issues is described in at
+    # https://yokota.blog/2021/08/26/understanding-protobuf-compatibility/
```

Comment:
Sorry I might be missing something here, I thought schema registry support BACKWARD, FORWARD, and FULL compatibility? The compatible check in the article seems to refer only to BACKWARD compatibility